sdks/python/apache_beam/testing/load_tests/load_test.py

```diff
@@ -31,8 +31,8 @@ def __init__(self):
     self.project_id = self.pipeline.get_option('project')
     self.metrics_namespace = self.pipeline.get_option('metrics_table')
     self._metrics_monitor = MetricsReader(
-        publish_to_bq=self.pipeline.get_option('publish_to_big_query') ==
-        'true',
+        publish_to_bq=self.pipeline.get_option('publish_to_big_query')
+        .lower() == 'true',
         project_name=self.project_id,
```

Comment:
My earlier suggestion would also help catch typos, since this flag is not declared a boolean.
If we end up removing test pipeline options, you could also use something like:
```
 parser.add_argument('--publish_to_big_query', default=False,
                      type=lambda s: bool(['false', 'true'].index(s.lower())),
```
It is easy to pass an incorrect value of a boolean flag without noticing the error...

Reply:
I suppose I hadn't understood your suggestion fully, my apologies. It's definitely worth implementing. I also added try/except block to make an exception a little more informative for users.

Reply:
Another way to make those options organized is to register them in `TestPipelineOption` (recommended if commonly used in performance testing), which works the same way as parser does.

Reply:
I created `LoadTestOptions` class with common parameters that load tests can take. `--publish_to_big_query` is one of them. I hope this will make those options better organised.